integration/utils/utils.go

```diff
@@ -0,0 +1,217 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"k8s.io/api/core/v1"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
```

Comment:
All these imports should be in the same block.

Reply:
fixed